### DISCUSSION TITLE
How to add Negex to config.cfg file?

### DISCUSSION THREAD

**Discussion by canberkarc** [explosion/spaCy#11165] [Help: Coding & Implementations] [answered]
There is no option to select Negex for config.cfg file on spaCy's official site(https://spacy.io/usage/training). How can I add Negex to config.cfg file? Thanks!

**Comment by polm** [comment, marked as answer]
I guess you're referring to [negspaCy](https://github.com/jenojp/negspacy)? That's not in the config generator because it's a third party project, we don't develop it. 

You can follow the instructions in their docs for adding a component, save that pipeline, and look at the generated config for an example of how to specify it.

**Reply by canberkarc** [reply]
Yes, I'm referring to negspaCy. I'll try that, thank you for your answer.
